org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -1309,3 +1309,54 @@ Sso8dxfh53Admin:
     instanceArn: !Ref instanceArn
     principalId: !Ref 8dxfh53AdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoRecoverProdAdmin:
+  Type: update-stacks
+  DependsOn: SsoAdministrator
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-recover-prod-admin'
+  StackDescription: 'SSO: Administrator role used by recover admin group'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref RecoverProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref recoverProdAdminGroup
```

Comment:
please add this to the `Parameters` section above 

```
  recoverProdAdminGroup:      #JC aws-recover-prod-admins
    Type: String
    Default: '54d83448-10a1-7000-7e72-cae28f0caacf'

  recoverProdDeveloperGroup:      #JC aws-recover-prod-developers
    Type: String
    Default: 'a468b478-f021-7010-535b-4ad779417559'

  recoverProdViewerGroup:      #JC aws-recover-prod-viewers
    Type: String
    Default: 'b4e86468-f091-70e2-0ea9-64cf3addcdf1'
```

Reply:
Done!

Reply:
sorry, i didn't mean add here.  The `Parameters:` section is from line 1 to  about 200.  Please add these parameters with somewhere in that section with the other JC group parameters.

Reply:
Done. Does this look correct now?